examples/material-vite/README.md

```diff
@@ -22,14 +22,14 @@ or:
 
 <!-- #default-branch-switch -->
 
-[![Edit on StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/github/mui/material-ui/tree/master/examples/vitejs)
+[![Edit on StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/github/mui/material-ui/tree/master/examples/material-vite)
 
-[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/mui/material-ui/tree/master/examples/vitejs)
+[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/mui/material-ui/tree/master/examples/material-vite)
 
 ## The idea behind the example
 
 This example uses [Vite.js](https://github.com/vitejs/vite).
-It includes `@mui/material` and its peer dependencies, including `emotion`, the default style engine in Material UI v5.
+It includes `@mui/material` and its peer dependencies, including Emotion, the default style engine in Material UI v5.
```

Comment:
```suggestion
It includes `@mui/material` and its peer dependencies, including [Emotion](https://www.npmjs.com/package/@emotion/react), the default style engine in Material UI v5.
```

or

```suggestion
It includes `@mui/material` and its peer dependencies, including [@emotion/react](https://www.npmjs.com/package/@emotion/react), the default style engine in Material UI v5.
```

If you agree with the suggestion, as you may already know, there are several other occurrences of this same sentence.

Reply:
Good call! I'll add a link wherever this comes up.